packages/expo-cli/src/exp.ts

```diff
@@ -334,7 +334,7 @@ Command.prototype.asyncAction = function (asyncFn: Action) {
     if (!getenv.boolish('EAS_BUILD', false) && !program.nonInteractive) {
       try {
         await profileMethod(checkCliVersionAsync)();
-      } catch (e) {}
+      } catch (e: any) {}
```

Comment:
IIRC you can just do `catch {}`. `(err)` seems to be optional since some recent TS version.

Reply:
I remember reading about it, but I just did a search and replace to make sure they were all typed (several hundreds of them), I didn't go over them to see if `e/err/error` was actually being used.